python/main.py

```diff
@@ -1,11 +1,29 @@
 import os
 import logging
 import pathlib
-from fastapi import FastAPI, Form, HTTPException
+import json
+import hashlib
+import sqlite3
+from fastapi import FastAPI, Form, UploadFile, HTTPException
 from fastapi.responses import FileResponse
 from fastapi.middleware.cors import CORSMiddleware
 
 app = FastAPI()
+
+conn=sqlite3.connect(dbname)
+cur=conn.cursor()
+
+
+cur.execute(CREATE TABLE items (
```

Comment:
There are a lot of syntax errors that mshibuya [mentioned](https://github.com/utaash88/mercari-build-training/pull/2#issuecomment-1960699118). Do you see them when you run `uvicorn`?